.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Report a bug in tutor-contrib-webhook-receiver
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Summary
+
+Replace this with a short summary (one sentence or short paragraph) of the issue you are seeing.
+
+## What I did
+
+Explain the steps you undertook that led to the unexpected behavior you are observing. Please include relevant documentation snippets, such as the `WEBHOOKRECEIVER__*` settings from Tutor’s `config.yml`.
```

Comment:
I think that's one underscore too many. :)

Reply:
It is indeed! :)